src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -0,0 +1,122 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import noop from 'noop-ts';
+import { Typography } from '@mui/material';
+import {
+  FormikTokenTextField,
+  LabeledProgressBar,
+  FormikSubmitButton,
+  ConnectWallet,
+  Icon,
+} from 'components';
+import { useTranslation } from 'translation';
+import useConvertToReadableCoinString from 'utilities/useConvertToReadableCoinString';
+import { format } from 'utilities/common';
+import { ConvertVrtForm } from '../Form';
+import { VRT_ID, XVS_ID } from '../constants';
+import { useStyles } from '../styles';
+
+interface ConvertProps {
+  xvsTotal: BigNumber;
+  xvsToVrtRate: BigNumber;
+  vrtLimitUsed: number;
+  vrtLimit: number;
```

Comment:
`vrtLimitUsed` and `vrtLimit ` need to use `BigNumber`, JavaScript numbers don't handle big values well.
Also, to be consistent with what we've done so far I'd add the units to each value (for example in this case I don't know if vrtLimitUsed is a percentage or a value in wei or VRT).

Reply:
I'll check on the units when I wire them up